app/controllers/RatePlanController.scala

```diff
@@ -50,7 +52,12 @@ class RatePlanController(
         paperPlans.voucher.everydayplus -> "Voucher Everyday+"
       ).map(tupleToRatePlanMap)),
       // TODO - Guardian Weekly - Quarterly, Annual, International Quarterly, International Annual
```

Comment:
Maybe rename this TODO to include the Zone A and Zone B - 'Guardian Weekly 1 Year' rate plans